deepchem/models/tests/test_reload.py

```diff
@@ -288,8 +288,6 @@ def test_normalizing_flow_model_reload():
   import tensorflow_probability as tfp
   tfd = tfp.distributions
   tfb = tfp.bijectors
-  tfk = tf.keras
-  tfk.backend.set_floatx('float64')
```

Comment:
In reload test of the normalizing flows, these lines sets all float values as `float64 ` in all tests after this test. I think these leads to failure of many tests.